demos/classification_benchmark_demo/cpp/grid_mat.hpp

```diff
@@ -129,11 +143,13 @@ class GridMat {
     static constexpr double fontScale = 1.5;
     static const int thickness = 2;
     static const int textPadding = 10;
-    static const std::string testMessage;
+    static constexpr const char testMessage[] = "Testing, please wait...";
     int baseline;
     cv::Size textSize;
     cv::Size accuracyMessageSize;
     cv::Size testMessageSize;
 };
 
-const std::string GridMat::testMessage = "Testing, please wait...";
+// defenition of the static member needed to remove clang build error
+// https://stackoverflow.com/questions/8016780/undefined-reference-to-static-constexpr-char
```

Comment:
```suggestion
```
I don't think you need to explain this. That's how the language works.

Reply:
gcc for example is fine without this definition, so I thought this is clang behavior only.

Reply:
It could be some kind of compiler extension enabled by default. But ok

Reply:
I already removed)

Reply:
Whatever